requirements.txt

```diff
@@ -23,3 +23,4 @@ pydocstyle~=1.0.0
 cmakelint~=1.3.4
 requests_mock~=0.7.0
 vim-vint~=0.3.6
+nltk
```

Comment:
In case nltk makes a shiny new version next year, coala users who are using coala 0.7.0 (which expects nltk's version X) would be installing the X+1 version of nltk from pip.
Hence it'd never work. So, yeah - version is necessary.

Note that very rare change is not quite true ... It doesnt harm to use the `~=` syntax. so why not use it ?